.buildkite/pipeline.yml

```diff
@@ -1,5 +1,51 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  SETUP_GVM_VERSION: 'v0.5.0'
+  GO_VERSION: "1.19.10"
+  GO_VERSION_CHOCO: "1.20.1"
```

Comment:
Is it possible to take Go version from https://github.com/elastic/elastic-agent-system-metrics/blob/main/.go-version ?

It's very important to maintain exactly the same version of Go as specified in this file. We have automation that updates the version when we're ready for the transition, some of minor version updates require us to make some changes to the code.

Reply:
It just tests because I don't see the 1.19.10 version as supported here: https://community.chocolatey.org/packages/golang#versionhistory
Also, I asked about it below on that page...

Reply:
I used this approach:
https://github.com/elastic/elastic-agent-system-metrics/blob/cf8c16fbbbb4aaf30c635f693702ffcb30490bc3/.buildkite/hooks/post-checkout#L5
For Windows tests, I used the 1.20.5 version of Golang. 
This line should be changed after migration of the repository to Go v.1.20:
https://github.com/elastic/elastic-agent-system-metrics/blob/cf8c16fbbbb4aaf30c635f693702ffcb30490bc3/.buildkite/scripts/run-win-tests.ps1#L33
to:
`withGolang $env:GO_VERSION`
and remove this one: 
https://github.com/elastic/elastic-agent-system-metrics/blob/cf8c16fbbbb4aaf30c635f693702ffcb30490bc3/.buildkite/pipeline.yml#L5